fvm/src/gas/outputs.rs

```diff
@@ -48,7 +48,7 @@ impl GasOutputs {
 
         if out.gas_burned != 0 {
             out.over_estimation_burn = base_fee_to_pay * out.gas_burned;
-            out.miner_penalty += (base_fee - base_fee_to_pay) * out.gas_burned;
```

Comment:
Implement https://doc.rust-lang.org/std/ops/trait.AddAssign.html

Reply:
I know I could have, I chose not to for simplicity. Will you require it?

Reply:
It's not _necessary_, but:

1. People expect it.
2. It can help with performance by reusing the allocation.

But we always add it in later.